packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -143,7 +143,10 @@ ln -sv %{_tmppath} %{buildroot}%{_datadir}/%{name}/tmp
 %license LICENSE
 %{_datadir}/%{name}
 %{_libexecdir}/%{name}/puppet_sign.rb
-%config(noreplace) %{_sysconfdir}/%{name}
+%dir %{_sysconfdir}/%{name}
```

Comment:
This is my attempt at avoid the warning that it listed the file twice. Not sure if this is needed though if we list files in that directory.